tools/styles/spectrum-scale-large.css

```diff
@@ -273,4 +273,7 @@
     --spectrum-dialog-confirm-description-text-size: var(
         --spectrum-global-dimension-font-size-75
     );
+    --spectrum-dialog-confirm-padding: var(
```

Comment:
May I know how did this new css variable got added?

Reply:
This was unintentional, will remove.